test/util.ts

```diff
@@ -18,6 +20,20 @@ export function expectFeeEstimationStructure(fee: any) {
     expect(typeof fee.gas_usage).to.equal("bigint");
 }
 
+export function expectStarknetPluginError(err: Error) {
+    expect(err).to.be.instanceOf(StarknetPluginError);
+}
+
+export function expectStarknetPluginErrorEqual(err: Error, expectedMessage: string) {
+    expect(err).to.be.instanceOf(StarknetPluginError);
+    expect(err.message).to.equal(expectedMessage);
+}
+
+export function expectStarknetPluginErrorContain(err: Error, expectedMessage: string) {
+    expect(err).to.be.instanceOf(StarknetPluginError);
+    expect(err.message).to.equal(expectedMessage);
```

Comment:
I think you copy-pasted from the other method.

Reply:
not my day :|

Reply:
fixed